src/python/pants/backend/python/tasks2/python_binary_create.py

```diff
@@ -103,6 +103,12 @@ def _create_binary(self, binary_tgt, results_dir):
 
       builder = PEXBuilder(path=tmpdir, interpreter=interpreter, pex_info=pex_info, copy=True)
 
+      if type(binary_tgt) is PythonBinary and binary_tgt.shebang:
+        self.context.log.info("Found Python binary target with customized shebang, using it: {}".format(binary_tgt.shebang))
+        builder.set_shebang(binary_tgt.shebang)
+      else:
+        self.context.log.warn('More than one Python binary target found or incompatible binary type. Customized Shebang will not be used!')
```

Comment:
The `_create_binary` function runs in the context of exactly 1 `binary_tgt` so the more-than-one language is not appropriate and the ramining language does not merit a warning - its _ok_ for a `PythonBinary` to not specify a `shebang`. So this log needs language change and demotion to `info` or `debug`.

Reply:
Adapted this to have more useful and appropriate output.